service/build.gradle

```diff
@@ -13,21 +15,20 @@ project.dependencies{
     compile(project(":persistence"))
 }
 
+bootJar {
+    enabled = false
+}
+
+jar {
+    enabled = true
+}
+
 dependencies {
-    // https://mvnrepository.com/artifact/org.mockito/mockito-all
-    testImplementation group: 'org.mockito', name: 'mockito-all', version: '1.10.19'
-    implementation group: 'org.springframework', name: 'spring-core', version: '5.3.5'
-    implementation group: 'org.springframework', name: 'spring-context', version: '5.3.5'
-    implementation group: 'org.springframework', name: 'spring-jdbc', version: '5.3.5'
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
```

Comment:
Do you need this dependency on the service layer?

Reply:
Yes, because In service layer used annotation @Transactional from org.springframework.transaction